torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -1341,5 +1354,11 @@ void FuseGraph(std::shared_ptr<Graph>& graph) {
   }
 }
 
+void CustomFuseGraph(std::shared_ptr<Graph>& graph, std::function<bool(Node*)> fn, Symbol kind) {
+  if (canFuseOnCPU() || canFuseOnGPU()) {
```

Comment:
My previous comment still applies: why do we check the capabilities of PyTorch fuser, when you're really applying this pass to obtain a fused node that will be passed through a completely different backend.